ansible/roles/ci/code_style_check/python/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Playbook for Python code style check
+
+- name: check if project_sources_path variable was set
+  fail: msg="Variable project_sources_path was not set"
+  when: project_sources_path is not defined
+
+- name: Read all packages paths in project
```

Comment:
There is small amount of duplication in the code. Lets leave it for now in order to get something working and I'll do refactoring after we get to some substantial functionality.